tests/flow/test_temporal.py

```
@@ -0,0 +1,482 @@
+                      d.dayOfQuarter, d.ordinalDay, d.hour, d.minute, d.second"""
+
+        res = self.graph.query(q).result_set
+        print(f"res: {res[0]}")
+
+        year         = res[0][0]
+        quarter      = res[0][1]
+        month        = res[0][2]
+        week         = res[0][3]
+        day          = res[0][4]
+        dayOfWeek    = res[0][5]
+        dayOfQuarter = res[0][6]
```

Ruff (0.11.9)

Local variable `dayOfQuarter` is assigned to but never used

Remove assignment to unused variable `dayOfQuarter`

(F841)